SyncExtension/UseCases/Files/UpdateFileContentUseCase.swift

```diff
@@ -143,7 +143,8 @@ struct UpdateFileContentUseCase {
                     bucketId: user.bucket,
                     progressHandler:{ completedProgress in
                         progress.completedUnitCount = Int64(completedProgress * 100)
-                    }
+                    },
+                    debug: true
```

Comment:
Should this be set to true always?

Reply:
nope, just for development purposes, removing it